credentials/src/main/kotlin/web5/sdk/credentials/PresentationExchange.kt

```diff
@@ -161,12 +164,16 @@ public object PresentationExchange {
     vcJwtList: Iterable<String>,
     presentationDefinition: PresentationDefinitionV2
   ): Map<InputDescriptorV2, List<String>> {
-    val vcJwtListWithNodes = vcJwtList.zip(vcJwtList.map { vcJwt ->
-      val vc = JWTParser.parse(vcJwt) as SignedJWT
+    val vcJwtListWithNodes = vcJwtList.zip(
+      vcJwtList.map { vcJwt ->
+        val vc = Jwt.decode(vcJwt)
 
-      JsonPath.parse(vc.payload.toString())
-        ?: throw JsonPathParseException()
-    })
+        val jwtModule = SimpleModule().addSerializer(JwtClaimsSet::class.java, JwtClaimsSetSerializer())
+        Json.jsonMapper.registerModule(jwtModule)
```

Comment:
Looks like we create and register the same `jwtModule` on every iteration of the `.map`. Maybe move this above the `.map` so we only do it once?

Reply:
great call, thanks!